protocol-spec/input.md

```diff
@@ -0,0 +1,107 @@
+# Generic Sensor Messages
+
```

Comment:
Like I said earlier, I'd really like to hold off on accellerometer until we've got more use cases for it. I realize it's in the vstroker/lovense/wiimote/etc, and if there's already normalization code across all of these in Buttplug C# maybe that's ok, but otherwise I feel like we're jumping the gun a bit here.

Reply:
Just looked up the code in buttplug C#, and the patches there seem to be sending raw accelerometer values from toys in this message. That means the user will not have any idea about the range limits unless they know the toy and its specs, which doesn't seem to help much. I'd really really really rather leave this post 0.2.